package.json

```
@@ -63,8 +63,10 @@
     "@stoplight/json-ref-resolver": "1.x.x",
     "@stoplight/types": "4.1.x",
     "@stoplight/yaml": "2.x.x",
+    "@types/js-yaml": "^3.12.1",
```

```suggestion
    "@types/js-yaml": "3.12.x",
```

;) that's the SL convention

---

Good to know, anyway it is not needed anymore so I removed it.